prog/setup_hugepages.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+class Prog::SetupHugepages < Prog::Base
+  subject_is :sshable, :vm_host
+
+  def start
+    @waited = 0
+    hugepage_size = "1G"
+    # put away 8G for host, try to reserve rest as hugepages
+    hugepage_cnt = vm_host.total_mem_gib - 8
```

Comment:
What happen if total_mem_gib is smaller than 8GB?

Reply:
We need to figure out what formula we need to use here. Anyway, I changed it so it is always positive.